examples/hypervisor_manager.tf

```diff
@@ -1,32 +1,70 @@
 provider "oneview" {
-	ov_username = "<username>"
-	ov_password = "<password>"
-	ov_endpoint = "<endpoint>"
-	ov_sslverify = false
-	ov_apiversion = <ov-apiversion>
-	ov_ifmatch = "*"
+  ov_username =   "${var.username}"
+  ov_password =   "${var.password}"
+  ov_endpoint =   "${var.endpoint}"
+  ov_sslverify =  "${var.ssl_enabled}"
+  ov_apiversion = 2200
```

Comment:
Take care of this as venkatesh mentioned in the other PR

Reply:
apiversion is an integer value, and HCL do not support integer data type for TF 1.11 version, since we are planning to upgrade, will incorporate this change in next release.